profiler/src/main/java/com/navercorp/pinpoint/profiler/context/exception/DisableExceptionRecordingServiceFactory.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2024 NAVER Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.navercorp.pinpoint.profiler.context.exception;
+
+import com.navercorp.pinpoint.profiler.context.exception.disabled.DisabledExceptionRecordingService;
+import com.navercorp.pinpoint.profiler.context.id.TraceRoot;
+
+public class DisableExceptionRecordingServiceFactory implements ExceptionRecordingServiceFactory {
```

Comment:
Maybe a typo: DisableExceptionRecordingServiceFactory -> Disable***d***ExceptionRecordingServiceFactory

ref)
`DisabledExceptionRecordingService`
`DisabledExceptionRecordingServiceFactoryProvider`

Reply:
Fixed it.